bin/sp-data-pull

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'json'
+
+data_file = 'your-json-file-here.json'
+
+imported_data = File.read(data_file)
+
+parsed_data = JSON.parse(imported_data)
+
+user_uuids = parsed_data.map { |user| user[:"properties.user_id"] }
```

Comment:
I'n guessing there is a corresponding query for Cloudwatch? if we're running on a worker host we would have access to the Cloudwatch insights API and could run this on a remote box in prod

Reply:
There isn't a specific query on this because might want to look at different events, but still only want user information.